k-distribution/src/main/scripts/lib/k-util.sh

```diff
@@ -11,12 +11,74 @@
 #
 # source "$(dirname "$0")/../lib/kframework/k-util.sh"
 
+# initialize flags
+fold_lines='fold -s'
+profile=false
+verbose=false
+
+# initialize state
+result=0
+
+error () {
+  local result
+  result="$1" ; shift
+  printf "[Error] Critical: $*\n" | ${fold_lines} 1>&2
+  exit ${result}
+}
+
+k_util_usage() {
+    cat <<HERE
+  --no-exc-wrap     Do not wrap messages to 80 chars (keep long lines).
+  --profile         Print coarse process timing information
```

Comment:
Can you add a short description of what those numbers mean?
`command time --format '%x %Es %Us %Ss %C' -- "$@"`
I ran this, but those numbers don't represent anything to me right now. and for sure I will forget in the future:
```
regression-new/issue-946$ krun --no-exc-wrap  -d . --profile
0 0:00.00s 0.00s 0.00s llvm-krun --directory /home/radu/work/k/k-distribution/tests/regression-new/issue-946/test-kompiled --dry-run -nm -o tmp.in.VQgZct4unN
```

Reply:
Done.

Reply:
Looks great.